packages/cli/src/generator.ts

```diff
@@ -234,93 +234,102 @@ export function getNamedImportNodes(
     return node;
   }
 
-  const importNames = getImports(
-    node.moduleSpecifier.text,
-    system,
-    sourceFile.fileName,
-    namedBindings.elements,
-  );
+  try {
```

Comment:
This diff is a bit messy, but essentially the only change here is the addition of a `try-catch` block and a change in indentation.